### PR TITLE
fix: Misc. spell cost ID fixes

### PR DESCRIPTION
**Pull request by P0keDev**

### CODE REVIEW

File: common/src/main/java/com/wynntils/core/webapi/profiles/item/IdentificationProfile.java [hunk 1]
```diff
@@ -32,14 +32,14 @@ public IdentificationProfile(IdentificationModifier type, int baseValue, boolean
     }
 
     public void calculateMinMax(String shortId) {
+        isInverted = IdentificationOrderer.INSTANCE.isInverted(shortId);
```
Comment (magicus) [inline common/src/main/java/com/wynntils/core/webapi/profiles/item/IdentificationProfile.java:35]: Why did you move this?

File: common/src/main/java/com/wynntils/wynn/objects/SpellType.java [hunk 2]
```diff
@@ -14,7 +14,7 @@ public enum SpellType {
     ARROW_SHIELD(ClassType.Archer, 4, "Arrow Shield", 8, 1),
 
     SPIN_ATTACK(ClassType.Assassin, 1, "Spin Attack", 6, 0),
-    VANISH(ClassType.Assassin, 2, "Vanish", 2, 0),
+    VANISH(ClassType.Assassin, 2, "Dash", 2, 0),
```
Comment (magicus) [inline common/src/main/java/com/wynntils/wynn/objects/SpellType.java]: Has the spell changed name in 2.0? Maybe we should update the name of the enum as well?

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/wynn/objects/SpellType.java]: It did, and I agree

File: common/src/main/java/com/wynntils/core/webapi/profiles/item/IdentificationProfile.java (hunk 1 above)
Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/core/webapi/profiles/item/IdentificationProfile.java:35]: the value was used earlier than calculated

Comment (magicus) [reply common/src/main/java/com/wynntils/core/webapi/profiles/item/IdentificationProfile.java:35]: Is it, really? Am I blind? It has moved from line 41 to line 35. I see no mention of `isInverted` between those lines. It is refered to in line 43, but the old position was before that as well. There are no method calls between line 35 and 41. I can't see how this change affects anything.

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/core/webapi/profiles/item/IdentificationProfile.java:35]: Line 40 returns early, so isInverted needs to be set before

Comment (magicus) [reply common/src/main/java/com/wynntils/core/webapi/profiles/item/IdentificationProfile.java:35]: Ah, now I get it. I was just really thick. Thanks!

File: common/src/main/java/com/wynntils/wynn/objects/SpellType.java (hunk 2 above)
Comment (P0keDev) [reply common/src/main/java/com/wynntils/wynn/objects/SpellType.java]: woops, good point

**Review by kristofbolyai — APPROVED** [review]
